Sources/OktaAnalytics/OktaAnalytics.swift

```diff
@@ -287,6 +302,33 @@ extension OktaAnalytics {
         }
     }
 
+    /**
+        Returns in completion `[ScenarioEvent]` objects that were created the specified number of seconds ago, if exists.
+
+        - Parameters:
+           - shouldBeDeleted: Delete fetched events and properties from the storage. The default value is true.
+        */
+    public static func getExpiredScenarioEvents(shouldBeDeleted: Bool = true,
```

Comment:
We don't need this function actually, we need something called reportAndDeleteExpiredEvents function.